### PR TITLE
drivers: sensor: Add SI7060 temperature sensor driver

### PR DESCRIPTION
**Pull request by iosif-macesanu**
This pull request adds support for the Silicon Labs SI7060 temperature sensor

### CODE REVIEW

File: drivers/sensor/si7060/si7060.c [hunk 1]
```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2019 Actinius
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <i2c.h>
+#include <sensor.h>
```
Comment (MaureenHelm) [inline drivers/sensor/si7060/si7060.c]: These should be:
```c
#include <drivers/i2c.h>
#include <drivers/sensor.h>
```

File: drivers/sensor/si7060/si7060.c [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+			.len = 1,
+			.flags = I2C_MSG_READ,
+		},
+	};
+
+	if (send_stop) {
+		msgs[1].flags |= I2C_MSG_STOP;
+	}
+
+	if (i2c_transfer(drv_data->i2c_dev, msgs,
```
Comment (MaureenHelm) [inline drivers/sensor/si7060/si7060.c]: `i2c_reg_read_byte` would be better here, as you would not have to manually construct the msgs array

File: drivers/sensor/si7060/si7060.c [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+	return 0;
+}
+
+static int si7060_reg_write(struct si7060_data *drv_data, u8_t reg,
+			      u8_t val)
+{
+	u8_t tx_buf[2] = {reg, val};
+
+	return i2c_write(drv_data->i2c_dev, tx_buf, sizeof(tx_buf),
```
Comment (MaureenHelm) [inline drivers/sensor/si7060/si7060.c]: Use `i2c_reg_write_byte` instead

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: drivers/sensor/si7060/si7060.c (hunk 1 above)
Comment (iosifmacesanu) [reply drivers/sensor/si7060/si7060.c]: Updated.

File: drivers/sensor/si7060/si7060.c (hunk 2 above)
Comment (iosifmacesanu) [reply drivers/sensor/si7060/si7060.c]: Updated.

File: drivers/sensor/si7060/si7060.c (hunk 3 above)
Comment (iosifmacesanu) [reply drivers/sensor/si7060/si7060.c]: Updated.

**Comment by iosifmacesanu** [comment]
Thanks for your suggestions @MaureenHelm

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**